### PR TITLE
1369: Show reservation count

### PR DESCRIPTION
**Pull request by xendk**
Updates FBS to use v2 API and adds the reservation count to the provider response.

### CODE REVIEW

File: modules/fbs/src/ExternalPatronApi.php [hunk 1]
```diff
@@ -122,7 +122,8 @@ public function getPreAuthenticatedPatron($agencyid, $cprNumber)
      *
      *
      *  The returned patron details includes a patronId that has to be used by all subsequent
-     *  service calls made on behalf of that patron.
+     *  service calls made on behalf of that patron. Note: Whis method can only be used for patrons who
```
Comment (guddo) [inline modules/fbs/src/ExternalPatronApi.php]: Typo

Comment (guddo) [inline modules/fbs/src/ExternalPatronApi.php]: Typo

File: modules/fbs/src/ExternalCatalogApi.php [hunk 2]
```diff
@@ -32,56 +33,24 @@ public function getAvailability($agencyid, $recordid)
     }
 
     /**
-     * Retrieves all booking information for each branch of an agency for bookings that ends after the current date.
```
Comment (guddo) [inline modules/fbs/src/ExternalCatalogApi.php]: Can you enlighten about this removal?

File: modules/fbs/src/ExternalPatronApi.php (hunk 1 above)
Comment (guddo) [inline modules/fbs/src/ExternalPatronApi.php]: Typo

File: modules/fbs/src/ExternalCatalogApi.php (hunk 2 above)
Comment (xendk) [inline modules/fbs/src/ExternalCatalogApi.php]: You would have to ask Systematic about that. It's been removed from the API.

**Comment by xendk** [comment]
Typos should be reported to Systematic, as the comments are generated from the API spec descriptions.

**Comment by guddo** [comment]
Can you rename your PR from "1396" to 1369"?

**Comment by JrgenGNielsen** [comment]
reviewed and approved

**Pull request merged**